internal/impl/qdrant/output.go

```diff
@@ -42,32 +42,32 @@ func outputSpec() *service.ConfigSpec {
 		Description(service.OutputPerformanceDocs(true, true)).
 		Fields(
 			service.NewOutputMaxInFlightField(),
-			service.NewBatchPolicyField(poFieldBatching),
-			service.NewStringField(poFieldGrpcHost).
+			service.NewBatchPolicyField(qoFieldBatching),
+			service.NewStringField(qoFieldGrpcHost).
 				Description("The gRPC host of the Qdrant collection. Defaults to 'localhost:6334'.").Default("localhost:6334"),
-			service.NewStringField(poFieldAPIToken).
+			service.NewStringField(qoFieldAPIToken).
 				Secret().
 				Description("The Qdrant API token for authentication. Defaults to an empty string.").Default(""),
-			service.NewTLSToggledField(poFieldUseTLS).Default("TLS(HTTPS) config to use when connecting"),
-			service.NewInterpolatedStringField(poFieldCollectionName).
-				Description("The name of the collection in Qdrant. REQUIRED"),
-			service.NewBloblangField(poFieldID).
-				Description("The ID of the point to insert. REQUIRED").
+			service.NewTLSToggledField(qoFieldUseTLS).Default("TLS(HTTPS) config to use when connecting"),
+			service.NewInterpolatedStringField(qoFieldCollectionName).
+				Description("The name of the collection in Qdrant."),
+			service.NewBloblangField(qoFieldID).
+				Description("The ID of the point to insert. Can be a UUID string or positive integer.").
 				Example(`root.id = this.id`).
```

Comment:
this should be `root = this.id`

Reply:
We're reading from the "id" key.
https://github.com/Anush008/connect/blob/main/internal/impl/qdrant/point_id.go

Setting `root` directly to a string, kept failing with some cryptic errors.

Reply:
```
root = "dc88c126-679f-49f5-ab85-04b77e8c2791"
```
Kept failing.

Reply:
It needed `QueryValue` instead of just `Query`. I pushed a fix

Reply:
Awesome. Thank you 🙏.